### PR TITLE
Export non-bundled module

### PR DESCRIPTION
**Pull request by tomalec**

### CODE REVIEW

**Comment by warpech** [comment]
@tomalec, maybe you'd like to review my changes in this PR: https://github.com/Starcounter-Jack/JSON-Patch/compare/b4b9de77d09e732f154c1784377bd9394ce99996...aa3e1db2761102e67f8700a793ef3f891b787092

**Comment by tomalec** [comment]
@warpech Would you like to re-review it, as we pushed quite a few commits, since your last approval, or is it good to go?

File: README.md [hunk 1]
```diff
@@ -24,195 +22,70 @@ JSON Patch plays well with the HTTP PATCH verb (method) and REST style programmi
...
-`jsonpatch.applyOperation` accepts a single operation object instead of a sequence, and returns the object after applying the operation. It works with all the standard JSON patch operations (`add, replace, move, test, remove and copy`).
-
-```js
-var document = { firstName: "Albert", contactDetails: { phoneNumbers: [] } };
-var operation = { op: "replace", path: "/firstName", value: "Joachim" };
-document = jsonpatch.applyOperation(document, operation).newDocument;
-// document == { firstName: "Joachim", contactDetails: { phoneNumbers: [] }}
+```html
+<script type="module">
+  import { * as jsonpatch } from 'fast-json-patch/index.mjs';
```
Comment (warpech) [inline README.md]: Change to `import * as jsonpatch from`

File: README.md [hunk 2]
```diff
@@ -24,195 +22,70 @@ JSON Patch plays well with the HTTP PATCH verb (method) and REST style programmi
...
-document.firstName = "Albert";
-document.contactDetails.phoneNumbers[0].number = "123";
-document.contactDetails.phoneNumbers.push({ number:"456" });
-var patch = jsonpatch.generate(observer);
-// patch  == [
-//   { op: "replace", path: "/firstName", value: "Albert"},
-//   { op: "replace", path: "/contactDetails/phoneNumbers/0/number", value: "123" },
-//   { op: "add", path: "/contactDetails/phoneNumbers/1", value: {number:"456"}}
-// ];
+import { * as jsonpatch } from 'fast-json-patch/index.mjs';
```
Comment (warpech) [inline README.md]: Same as above

File: README.md [hunk 3]
```diff
@@ -24,195 +22,70 @@ JSON Patch plays well with the HTTP PATCH verb (method) and REST style programmi
...
-document.contactDetails.phoneNumbers.push({ number:"456" });
-var patch = jsonpatch.generate(observer, true);
-// patch  == [
-//   { op: "test", path: "/firstName", value: "Joachim"},
-//   { op: "replace", path: "/firstName", value: "Albert"},
-//   { op: "test", path: "/contactDetails/phoneNumbers/0/number", value: "555-123" },
-//   { op: "replace", path: "/contactDetails/phoneNumbers/0/number", value: "123" },
-//   { op: "add", path: "/contactDetails/phoneNumbers/1", value: {number:"456"}}
-// ];
+import { * as jsonpatch } from 'fast-json-patch';
```
Comment (warpech) [inline README.md]: Same as above

**Comment by tomalec** [comment]
Released in https://github.com/Starcounter-Jack/JSON-Patch/releases/tag/3.0.0-0

**Pull request merged**